content/en/tracing/setup_overview/setup/dotnet-framework.md

```diff
@@ -42,9 +42,32 @@ further_reading:
     tag: "GitHub"
     text: "Source code"
 ---
+## Capabilities
+
+- Monitor [runtime metrics][1] to help troubleshoot bugs and detect resource inefficiencies in .NET applications & services.
+
+- Ability to correlate [traces & application logs][2].
+
+- End-to-end visibility with [Real User Monitoring][3] by linking requests from the web and mobile applications to their corresponding backend traces.
+
 ## Compatibility requirements
 
-The .NET Tracer supports automatic instrumentation on .NET Framework 4.5 and above. For a full list of supported libraries, visit the [Compatibility Requirements][1] page.
+### Supported .NET Framework runtimes
+The .NET Tracer supports instrumentation on .NET Framework 4.5 and above (using CLR v4.0).
```

Comment:
What if we follow the text from the PHP page so we don't have two places with compatibility tables:

```
For a full list of supported libraries and language versions, visit the [Compatibility Requirements][4] page.
```

Reply:
I think we should have one table on the Compatibility requirements page and just link back to that as needed

Reply:
I agree with Zach. The table doesn't look great (it's a simple list of versions so it takes up a lot of "empty" space) and it pushes the "Getting started" section under the fold.

![image](https://user-images.githubusercontent.com/1851278/127045194-9273e779-1fb5-4ef3-8d54-272e35e65339.png)

Reply:
Agree, will update.

Reply:
Updated, & removed reflected in: https://github.com/DataDog/documentation/pull/11200/commits/77793e5a94b644a2679fb90ab651acd9a154876b